db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_10_13_083210) do
+ActiveRecord::Schema[7.0].define(version: 2023_11_06_131223) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
```

Comment:
this should not change as there are no migrations

Reply:
yes i know, there was a migration, which i used wrongfully and then deleted it, evea already pointed it out, thats why i deleted the migration, but thanks for informing